packages/grid/x-data-grid/src/locales/frFR.ts

```diff
@@ -100,7 +100,7 @@ const frFRGrid: Partial<GridLocaleText> = {
       : `${count.toLocaleString()} ligne sélectionnée`,
 
   // Total row amount footer text
-  footerTotalRows: 'Lignes totales :',
+  footerTotalRows: 'Total de lignes :',
```

Comment:
This one is actually the one that made me do this PR. I think, it's just better like that.